src/App.tsx

```diff
@@ -1,155 +1,103 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
 
-const USER_ID = 0;
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+import { SelectTodo } from './types/SelectTodo';
+
+const USER_ID = 10586;
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState<string>('');
+  const [selected, setSelected] = useState<string>('All');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [hideError, setHideError] = useState<boolean>(false);
+
+  useEffect(() => {
+    const timerId = setTimeout(() => {
+      setHideError(false);
+    }, 3000);
+
+    return () => {
+      clearInterval(timerId);
+    };
+  }, []);
```

Comment:
It's false by default, why do you need to set it false again after 3 seconds??

Reply:
agree. Remove useless useEffect